### PR TITLE
added padding and changed the look of the description

### PR DESCRIPTION
**Pull request by Nickpopadopolous**

### CODE REVIEW

File: secondPage.html [hunk 1]
```diff
@@ -164,7 +164,7 @@ <h5 id="theatreName"></h5>
 
         		<div class="tabs-panel is-active" id="panel1">
 
-          			<h4>Description</h4>
+          			<h4></h4>
```
Comment (AdamJahur) [inline secondPage.html]: why did u leave the <h4> blank?

Comment (AdamJahur) [inline secondPage.html]: h4

**Comment by AdamJahur** [comment]
why is the h4 blank

**Comment by AdamJahur** [comment]
oops wrong button

File: secondPage.html (hunk 1 above)
Comment (Nickpopadopolous) [inline secondPage.html]: I found it repetitive to keep description on the tab and above the movie poster on the second page. I meant to delete the h4 if I liked it.

Comment (Nickpopadopolous) [inline secondPage.html]: We can leave it in, I don't care.

Comment (AdamJahur) [inline secondPage.html]: okay yea thats fine

**Comment by AdamJahur** [comment]
looks legit

**Comment by LibbyKnight** [comment]
Cool

**Comment by dzimmerman902** [comment]
Looks good to me

**Pull request merged**